conf/pacific/rados/6-node-cluster-6-clients.yaml

```diff
@@ -60,34 +49,50 @@ globals:
           - mgr
           - mds
           - rgw
-          - node-exporter
-          - crash
       node7:
+        image-name:
```

Comment:
Assuming this global conf is not part of any execution. So that it would not have any affect.

Reply:
Yes.